components/chat-drawer/ChatDrawer.tsx

```
@@ -22,9 +19,12 @@ function ChatDrawer() {
         <View className="flex-1 p-2">
             <Text className="p-4 text-lg font-bold text-slate-900 dark:text-slate-100">Chat history</Text>
             <ScrollView fadingEdgeLength={192}>
-                {chats.map(
-                    (chat) => <ChatElement chat={chat} key={chat.id} selected={chatId === chat.id}/>,
-                )}
+                <ChatsSection title="Today" startDay={0}/>
+                <ChatsSection title="Yesterday" startDay={1} endDay={2}/>
+                <ChatsSection title="Last 7 days" startDay={2} endDay={8}/>
+                <ChatsSection title="Last 14 days" startDay={8} endDay={15}/>
+                <ChatsSection title="Last month" startDay={15} endDay={31}/>
+                <ChatsSection title="Older" startDay={31} endDay={Infinity}/>
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)